test/hotel-test.js

```diff
@@ -0,0 +1,55 @@
+import chai from 'chai';
+const expect = chai.expect;
+import rooms from './test-data/rooms-data';
+import bookings from './test-data/bookings-data';
+import hotel from '../src/hotel';
+
+describe('hotel object', function() {
+  
+  beforeEach(function () {
+    hotel.roomInfo = rooms;
+    hotel.bookingInfo = bookings;
+  });
+
+  it('should store info about the rooms', function() {
+    expect(hotel.roomInfo).to.deep.equal(rooms);
+  });
+
+  it('should store info about bookings', function() {
+    expect(hotel.bookingInfo).to.deep.equal(bookings);
+  });
```

Comment:
What info are you storing here? Might be worth testing to ensure that all your properties are present.